src/ensembl/src/content/app/entity-viewer/EntityViewer.tsx

```diff
@@ -52,8 +53,9 @@ const EntityViewer = (props: Props) => {
       {params.entityId ? (
         <StandardAppLayout
           mainContent={<div>Main content is coming...</div>}
-          sidebarContent={<div>Sidebar content is coming...</div>}
-          sidebarNavigation={<EntityViewerSidebarTabs />}
+          sidebarContent={<EntityViewerSidebar />}
+          sidebarNavigation={<div>Sidebar navigation goes here</div>}
```

Comment:
`<EntityViewerSidebarTabs />` should stay.

Reply:
Ah good catch. Thanks!

Reply:
✔️